lib/request.js

```diff
@@ -0,0 +1,233 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Can we merge this file with the one on `master` currently under `lib/util/request.js`?

Reply:
Deleted. Not used.